employees/templates/employees/report_list.html

```diff
@@ -100,6 +100,9 @@ <h4><font color="red">{{ error }}</font></h4>
             <a href="{% url 'export-data-xlsx' pk=user.pk year=date.0 month=date.1 %}" class="btn btn-info">
                 Export {{ date.1 | convert_to_month_name }} {{ date.0 }} Reports
```

Comment:
Please add here that this gives `XLSX` export :)

Reply:
:+1: done